pom.xml

```diff
@@ -296,9 +296,9 @@
                 <version>3.8.1</version>
                 <configuration>
                     <outputFileName>TotalFreedomMod.jar</outputFileName>
-                    <compilerVersion>11</compilerVersion>
-                    <source>11</source>
-                    <target>11</target>
+                    <compilerVersion>1.8</compilerVersion>
```

Comment:
Why? We are now on Java 11, please change it back.

Reply:
i am using java 8 on my pc so i changed it to java 8. I will change back :D